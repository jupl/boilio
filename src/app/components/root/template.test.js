import React from 'react'
import renderer from 'react-test-renderer'
import Root from './template'

describe('<App.Root> Template', () => {
  it('should render as expected', () => {
    const component = renderer.create(<Root />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
