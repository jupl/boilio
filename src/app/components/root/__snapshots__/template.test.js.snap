exports[`<App.Root> Template should render as expected 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 24,
      }
    }>
    Welcome to 
    Test
  </Text>
</View>
`;
